components/choices/index.js

```diff
@@ -39,10 +39,10 @@ const updateChoiceById = async (req, response) => {
   let i = 0;
 
   try {
-    const values = getInputFromSigPayload(payloadBytes);
+    const { votesData } = getInputFromSigPayload(payloadBytes);
```

Comment:
Can you change this to what it was earlier so we don't do unnecessary changes in the code. I feel like this might break the code since there's a slight diffence in `const value` and `const { votesData}`

Reply:
fixed!